Autopilot/Src/SensorFusion.cpp

```diff
@@ -6,7 +6,13 @@
 #include "MadgwickAHRS.h"
 #include <math.h>
 
-SFError_t SF_GetResult(SFOutput_t *Output, IMU_Data_t *imudata, Airspeed_Data_t *airspeeddata) {
+typedef struct {
+    float roll, pitch, yaw; //rad
+    float rollRate, pitchRate, yawRate; //rad/s
+    float Airspeed; //m/s
```

Comment:
track was added into here some time ago. THis is rebasing problem shit. Uh Oh. We're gonna have a hell of a time merging this LOOOL but we'll cross that bridge when we come to it.

Don't bother doing anything about it now though.